modules/repository-azure/build.gradle

```diff
@@ -326,6 +327,12 @@ tasks.register("workloadIdentityYamlRestTest", RestIntegTestTask) {
   // omitting key and sas_token so that we use a bearer token from workload identity
 }
 
+if (BuildParams.inFipsJvm) {
+  // Cannot override the trust store in FIPS mode, and these tasks require a HTTPS fixture
+  tasks.named("managedIdentityYamlRestTest").configure { enabled = false }
+  tasks.named("workloadIdentityYamlRestTest").configure { enabled = false }
+}
```

Comment:
Since we disable these tests globally, I think we should remove the `ESTestCase.inFipsJvm()` in `RepositoryAzureClientYamlTestSuiteIT` and `AzureSnapshotRepoTestKitIT`? IIUC, we will always use HTTPS fixture when fixture is in use? If so, might also make sense to merge `USE_FIXTURE` and `USE_HTTPS_FIXTURE` into one. Can be a follow-up if you prefer.

Reply:
We run the regular `yamlRestTest` variants of `RepositoryAzureClientYamlTestSuiteIT` and `AzureSnapshotRepoTestKitIT` in FIPS mode too, and we want to use the HTTP fixture in those cases.